content/support/enablement/k8-migration.md

```diff
@@ -0,0 +1,101 @@
+# How to Migrate Sourcegraph from one cluster to another.
+
+The instructions in this document will guide you on the steps you need to take to migrate your Sourcegraph deployment from one Cluster to another. This is a process that involves the backing up of the Sourcegraph database by generating dumps and restoring the dumps in the newly deployed Sourcegraph instance on another cluster.
+
+> WARNING: **Only core data will be backed up**.
+>
+> These instructions will only back up core data including user accounts, configuration, repository-metadata, etc. Other data will be regenerated automatically:
+>
+> - Repositories will be re-cloned
+> - Search indexes will be rebuilt from scratch
+>
+> The above may take a while if you have a lot of repositories. In the meantime, searches may be slow or return incomplete results. This process rarely takes longer than 6 hours and is usually **much** faster.
+
+> NOTE: In some places you will see `$NAMESPACE` used. Add `-n $NAMESPACE` to commands if you are not using the default namespace
```

Comment:
Adding an example of where / what adding `-n $NAMESPACE` to the command's looks like and what that means would be helpful.